### PR TITLE
use modal dialog

### PR DESCRIPTION
**Pull request by bfalacerda**
needs https://github.com/strands-project/strands_ui/pull/106 and https://github.com/strands-project/strands_navigation/pull/341

Should solve the issue with the walking group. Also looks nicer

### CODE REVIEW

File: strands_human_help/src/strands_human_help/help_screen.py [hunk 1]
```diff
@@ -76,19 +68,34 @@ def being_helped(self, failed_component, interaction_service, n_fails):
...
-            buttons = [(label, interaction_service)]
-            content = strands_webserver.page_utils.generate_alert_button_page(html, buttons, self.service_prefix)
-        strands_webserver.client_utils.display_content(self.display_no, content)
+            goal = ModalDialogSrvGoal(title = "Please help",
+                                  text = html,
+                                  buttons = [label],
+                                  buttons_class = ['btn-success']
+                                  )
+        self.screen.send_goal(goal)
+        self.screen.wait_for_result()
```
Comment (marc-hanheide) [inline strands_human_help/src/strands_human_help/help_screen.py:95]: Does this also work when the goal is preempted, i.e. there is no result??

Comment (bfalacerda) [reply strands_human_help/src/strands_human_help/help_screen.py:95]: yes, another thread does the preemption when the timeout occurs, and this wait is released when that happens. The result is an empty msg in that case, that's why i then check if the button was pressed below

**Comment by marc-hanheide** [comment]
This appears to be working. You *could* decide to merge from my point of view, but this will require other merges, too, first.

**Comment by bfalacerda** [comment]
merging then, all needed merges are done

**Pull request merged**